### PR TITLE
Moved more traits to Mods.Common

### PR DESCRIPTION
**Pull request by Mailaender**
Tried to see how far I could come regarding https://github.com/OpenRA/OpenRA/issues/7526.

### CODE REVIEW

File: mods/ra/mod.yaml [hunk 1]
```diff
@@ -67,8 +67,6 @@ Chrome:
 Assemblies:
 	./mods/common/OpenRA.Mods.Common.dll
 	./mods/ra/OpenRA.Mods.RA.dll
-	./mods/cnc/OpenRA.Mods.Cnc.dll
```
Comment (pchote) [inline mods/ra/mod.yaml]: This will break the custom RA maps that port the Ion Cannon and other TD things into RA.

Comment (Mailaender) [inline mods/ra/mod.yaml]: Are there any?

Comment (Phrohdoh) [inline mods/ra/mod.yaml]: There are quite a few on the resource center, unfortunately.

Comment (Mailaender) [inline mods/ra/mod.yaml]: So keep that DLL or move the Ion Cannon to Mods.Common?

Comment (pchote) [inline mods/ra/mod.yaml]: My preference would be to keep the DLL for now, and then replace the overly specific Ion Cannon and Nuke support powers with a more general "Fire an armament" power in mods.common.

Comment (Mailaender) [inline mods/ra/mod.yaml]: Updated.

**Comment by obrakmann** [comment]
Looks ok to me, didn't notice any regression in-game. :+1: 

Needs a rebase.

**Comment by Mailaender** [comment]
Rebased.

File: OpenRA.Mods.RA/Scripting/Properties/ParadropProperties.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+using OpenRA.Mods.Common.Activities;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Mods.RA.Traits;
+using OpenRA.Scripting;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.RA.Scripting
+{
+	[ScriptPropertyGroup("Paradrop")]
+	public class ParadropPowers : ScriptActorProperties, Requires<CargoInfo>, Requires<ParaDropInfo>
```
Comment (penev92) [inline OpenRA.Mods.RA/Scripting/Properties/ParadropProperties.cs]: I'm not familiar with how the Lua API works, but shouldn't this be called `ParadropProperties` ?

Comment (obrakmann) [inline OpenRA.Mods.RA/Scripting/Properties/ParadropProperties.cs]: I agree with penev here, please rename for consistency, and then let's get this merged.

Comment (Mailaender) [inline OpenRA.Mods.RA/Scripting/Properties/ParadropProperties.cs]: Sorry. Forgot about this one. Updated.

**Comment by reaperrr** [comment]
Looks good and noticed no issues on RA and TD shellmaps. :+1:

**Pull request merged**